gradio/interpretation.py

```diff
@@ -198,7 +198,7 @@ def quantify_difference_in_label(interface, original_output, perturbed_output):
             score = diff(original_label, perturbed_label)
         return score
 
-    elif isinstance(output_component, Textbox):
+    elif isinstance(output_component, (Textbox, Number)):
```

Comment:
I think only having Number here would be enough, because we only support Text with Textbox and Number with Number.

Reply:
Makes sense.